hvac_services/data/res_config_settings.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="res_config_settings_enable" model="res.config.settings">
+        <field name="group_sale_order_template" eval="True"/>
+        <field name="group_stock_production_lot" eval="True"/>
+        <field name="group_project_recurring_tasks" eval="True"/>
+        <field name="module_industry_fsm_report" eval="True"/>
```

Comment:
Is it necessary? I guess if industry_fsm_sale_report is installed, this module is as well
```suggestion
```